pkgs/applications/editors/emacs/elisp-packages/update-from-overlay

```diff
@@ -1,26 +1,27 @@
-#!/usr/bin/env nix-shell
-#! nix-shell -i bash -p curl nix coreutils
+#!/usr/bin/env nix
+#! nix shell nixpkgs#bash nixpkgs#curl nixpkgs#nix nixpkgs#coreutils --command bash
```

Comment:
I do not think this is a good idea because it needs experimental features `nix-command` and `flakes` which users may not or do not want to set.

Reply:
OK, removed.